pkg/handlers/defaulthandler.go

```diff
@@ -110,14 +152,19 @@ func (h *defaultHandler) openArchive(ctx logContext.Context, depth int, reader i
 		// Decompress tha archive and feed the decompressed data back into the archive handler to extract any nested archives.
 		compReader, err := archive.OpenReader(arReader)
 		if err != nil {
-			return err
+			return fmt.Errorf("error opening decompressor with format: %s %w", format.Name(), err)
```

Comment:
this is minor but this format string looks like it might be unnecessarily difficult to quickly parse while debugging. i'd expect something more like `error opening decompressor with format %q: %w`